.circleci/config/jobs/@jobs.yml

```diff
@@ -1,6 +1,6 @@
 build_image:
   machine:
-    image: ubuntu-2004:202010-01
+    image: ubuntu-2004:202111-01
```

Comment:
can this be extracted into variable to be changed in 1 place?

Reply:
I couldn't make it work, CircleCI didn't like the syntax for some reason